md/wildfly-bundle.md

```diff
@@ -40,113 +40,128 @@ The Wildfly bundle is based on a standard Wildfly installation with the followin
+If this is a license update, use [the *Platform setup tool*](BonitaBPM_platform_setup.md#update_platform_conf).
 
-To do so, go to `<WILDFLY_HOME>/` and update the `setup/database.properties` files with the connection information of the  database.  
 
-<a id="database" />
+### Change the default credentials (optional, recommended for production)
 
-#### Configure the Wildfly server datasources
+As a security precaution, we **strongly recommend** that before you start your application server, you change the default username and password used for the platform administrator and for the default tenant administrator.
 
-After configuring the datasource to let the Platform setup tool intialize the configuration, you need to configure this datasource on the server. Follow those steps:
+#### Platform administrator
 
-1. Update the datasource configuration in `standalone/configuration/standalone.xml` by uncommenting and commenting relevant parts
-2. Update the `<property name="sysprop.bonita.db.vendor" value="h2" />` in `standalone/configuration/standalone.xml` with your database vendor
-3. Add the JDBC driver as decribed in the [database configuration](database-configuration.md#jdbc_driver) page
+The username and password for the platform administrator are defined in the file [`bonita-platform-community-custom.properties`](BonitaBPM_platform_setup.md), by the following properties:
 
+* `platformAdminUsername` defines the username (default `platformAdmin`)
+* `platformAdminPassword` defines the password (default `platform`)
 
-### Get and install a license
+This password is used for platform-level administration tasks, such as creating a tenant.
 
-::: info
-This is not necessary if you are installing the Community edition.
+#### Tenant administrator
+
+Each tenant has an administrator, with a tenant-specific username and password. The tenant administrator is also known as the tenant technical user.
+
+When the platform is created, default values for the tenant administrator username and password are defined in the file [`bonita-tenant-community-custom.properties`](BonitaBPM_platform_setup.md), by the following properties:
+
+* `userName` defines the username (default `install`)
+* `userPassword` defines the password (default `install`)
+
+When you create a tenant, the tenant administrator is created with the default username and password, unless you specify new values. 
+Change these tenant-specific credentials for an existing tenant by updating the `userName` and `userPassword` properties in `bonita-tenant-community-custom.properties`.
+
+::: warning
+For the default tenant, the tenant administrator username and password must also be defined in file [`platform-tenant-config.properties`](BonitaBPM_platform_setup.md), with exactly the same values that you set in `bonita-tenant-community-custom.properties`. 
+At platform creation, this file contains the default username and password for the default tenant. 
```

Comment:
Sad we have the exact same content than in tomcat bundle page.
Would it be accurate to create a specific page for this section and just link it in tomcat/wildfly bundles pages ?